ihmc-common-walking-control-modules/src/main/java/us/ihmc/commonWalkingControlModules/controlModules/foot/WorkspaceLimiterParameters.java

```diff
@@ -57,7 +60,7 @@ public WorkspaceLimiterParameters(YoRegistry parentRegistry)
 
       velocityDifferenceForLengthening = new DoubleParameter(namePrefix + "VelocityDifferenceForLengthening", registry, defaultMinVelocityDifference);
 
-      enableSingularityAvoidanceOnSwingFoot = new BooleanParameter(namePrefix + "enableSingularityAvoidanceOnSwingFoot", registry, true);
+      enableSingularityAvoidanceOnSwingFoot = new BooleanParameter(namePrefix + "enableSingularityAvoidanceOnSwingFoot", registry, false);
```

Comment:
Turning this off. The height module should lower the foot, and the touchdown velocity should force it to make contact. With this on, it has a tendency to float.